llvm/test/CodeGen/X86/atomic-scalarization.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: llc %s --mtriple=x86_64 -print-after=unpack-mi-bundles -disable-verify 2>&1 | FileCheck %s
```

Comment:
Use -stop-after, do not rely on -print-after output. It is NOT real mir. Also don't use disable-verify. Also just use finalize-isel

Reply:
Ah, will use -stop-after. thanks.. Atomic load with vector is not legal as per IR/Verifier, so we need disable-verify as far as I can tell.

Reply:
Then this shouldn't be handled at all